tests/integration/acp.go

```diff
@@ -356,7 +364,9 @@ func setupSourceHub(s *state) ([]node.ACPOpt, error) {
 		return nil, err
 	}
 
-	out, err := exec.Command("sourcehubd", "init", moniker, "--chain-id", chainID, "--home", directory).CombinedOutput()
+	args := []string{"init", moniker, "--chain-id", chainID, "--home", directory}
```

Comment:
question: Just a quick check - you intend to commit these logs to develop?  They weren't accidentally included in the PR?

Reply:
yeah, plan to keep it.
I didn't see much use of a command's output to console without the comman itself.